### ISSUE TITLE
[gtk3] Quick Open cosmetics

### ISSUE THREAD

**Issue by egmontkob** [Guake/guake#1120]
- It's truly hard to parse the bottom two lines, it's even unclear without studying the details that it's an arbitrary custom label, followed by a regex. It's pretty easy to believe that the label (like "Python traceback: ") is also part of the regex to match.

- The colon separator after a full sentence ending in a dot ("File path should exists.:") looks weird.

My suggestion: Instead of the multiline text entry field, maybe a treeview with two columns (and a header explaining them) would be much nicer.

- It's the bottom line here that determines the overall width of the Prefs window. Interestingly, a few pixels are always chopped off from its right. Change the label in Guake's source and try out: the Prefs window will be narrower or wider accordingly, and the last 2 or so characters of the regex will be stripped off. I suspect indent=15 in the glade file being the culprit, which should be converted to a margin_start, or left_padding of the parent or so.

- The label "line starts by" should begin with uppercase: "Line starts by"

- "should exists" should be "should exist". I don't see the point in this sentence, though.

- The list is not editable. I recommend to at least bring back the warning that the gtk2 version had here to contact the author to expand the list.

- The location of this list (below the "Quick open in current terminal" checkbox, without any header, and even indented) seriously suggests that it belongs to that checkbox, that is, only relevant if  Quick open in current terminal is enabled, which is not the case. This list should have its unindented header, and/or be moved above this checkbox.

**Comment by Stibbons** [comment]
quick open panel needs to be reworked, i agree with your idea of the tree or listview
